### PR TITLE
chore: add news guida software libero

### PR DESCRIPTION
**Pull request by libremente**
This PR:
* adds a news regarding the publication of the new guide  on free software development.

### CODE REVIEW

File: _posts/it/2020-12-29-guida-sviluppo-gestione-software-libero.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+### Chi l’ha scritta?
+
+La guida è frutto del lavoro di un anno delle esperienze del team di Developers Italia, del [Dipartimento della Trasformazione Digitale](https://innovazione.gov.it/it/chi-siamo/dipartimento/), per cristallizzare la conoscenza generata durante il lavoro di supporto alle Pubbliche Amministrazioni sia centrali che locali.
+
+### Chi sono i protagonisti?
+
+I protagonisti sono le Pubbliche Amministrazioni, e le buone pratiche che abbiamo imparato insieme attraverso le nostre attività di supporto. Per questo motivo la guida è strutturata secondo capitoli specifici dedicati proprio ai diversi profili professionali che potrebbero averne bisogno: decisori pubblici, funzionari pubblici e sviluppatori di servizi pubblici. E’ proprio a questi ultimi che si rivolge il primo capitolo che pubblichiamo oggi.
+
+### Perchè riteniamo sia utile?
```
Comment (bfabio) [inline _posts/it/2020-12-29-guida-sviluppo-gestione-software-libero.md:31]:
```suggestion
### Perché riteniamo sia utile?
```

File: _posts/it/2020-12-29-guida-sviluppo-gestione-software-libero.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ 
+La versione che pubblichiamo oggi, seppur revisionata e corretta, è solo una prima parte in forma di bozza: in perfetto stile “open source” la continueremo a sviluppare e migliorare collaborativamente e costantemente con i lettori e con tutti coloro che in futuro desidereranno offrire il loro contributo.
+
+### Chi l’ha scritta?
+
+La guida è frutto del lavoro di un anno delle esperienze del team di Developers Italia, del [Dipartimento della Trasformazione Digitale](https://innovazione.gov.it/it/chi-siamo/dipartimento/), per cristallizzare la conoscenza generata durante il lavoro di supporto alle Pubbliche Amministrazioni sia centrali che locali.
+
+### Chi sono i protagonisti?
+
+I protagonisti sono le Pubbliche Amministrazioni, e le buone pratiche che abbiamo imparato insieme attraverso le nostre attività di supporto. Per questo motivo la guida è strutturata secondo capitoli specifici dedicati proprio ai diversi profili professionali che potrebbero averne bisogno: decisori pubblici, funzionari pubblici e sviluppatori di servizi pubblici. E’ proprio a questi ultimi che si rivolge il primo capitolo che pubblichiamo oggi.
```
Comment (bfabio) [inline _posts/it/2020-12-29-guida-sviluppo-gestione-software-libero.md:29]:
```suggestion
I protagonisti sono le Pubbliche Amministrazioni, e le buone pratiche che abbiamo imparato insieme attraverso le nostre attività di supporto. Per questo motivo la guida è strutturata secondo capitoli specifici dedicati proprio ai diversi profili professionali che potrebbero averne bisogno: decisori pubblici, funzionari pubblici e sviluppatori di servizi pubblici. È proprio a questi ultimi che si rivolge il primo capitolo che pubblichiamo oggi.
```

File: _posts/it/2020-12-29-guida-sviluppo-gestione-software-libero.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Siamo consci del fatto che i temi legati allo sviluppo e alla manutenzione del software possono risultare complessi e alle volte anche oscuri a chi non si occupa di programmazione. Allo stesso modo, lavorare con soggetti pubblici in un contesto composto di adempimenti normativi e stringenti regole tecniche può risultare sfidante per uno sviluppatore (o per un fornitore) abituato al mondo del privato.
+
+Questo elaborato vuole dunque essere un supporto operativo sia per l’amministratore pubblico che per il tecnico, offrendo ad entrambi uno sguardo più ampio per evidenziare i vantaggi di costruire un ecosistema open source vitale.
```
Comment (bfabio) [inline _posts/it/2020-12-29-guida-sviluppo-gestione-software-libero.md:39]:
```suggestion
Questo elaborato vuole dunque essere un supporto operativo sia per l’amministratore pubblico che per il tecnico, offrendo a entrambi uno sguardo più ampio per evidenziare i vantaggi di costruire un ecosistema open source vitale.
```

**Review by sebbalex — APPROVED** [review]
:rocket:

File: _posts/it/2020-12-29-guida-sviluppo-gestione-software-libero.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+### Con quale spirito è stata stesa?
+
+Lo abbiamo già detto, ma ci piace sottolinearlo nuovamente: il nostro intento è quello di supportare concretamente ed operativamente le Pubbliche Amministrazioni a districarsi in un contesto nuovo e complesso, offrendo soluzioni pratiche apprese sul campo invece che liste di norme ed adempimenti. Queste soluzioni sono studiate, appunto, per massimizzare la semplicità e l’efficacia in termini di creazione di ecosistemi aperti.
```
Comment (bfabio) [inline _posts/it/2020-12-29-guida-sviluppo-gestione-software-libero.md:43]:
```suggestion
Lo abbiamo già detto, ma ci piace sottolinearlo nuovamente: il nostro intento è quello di supportare concretamente e operativamente le Pubbliche Amministrazioni a districarsi in un contesto nuovo e complesso, offrendo soluzioni pratiche apprese sul campo invece che liste di norme e adempimenti. Queste soluzioni sono studiate, appunto, per massimizzare la semplicità e l’efficacia in termini di creazione di ecosistemi aperti.
```

File: _posts/it/2020-12-29-guida-sviluppo-gestione-software-libero.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+In altre parole, in questa guida abbiamo provato a fondere la nostra conoscenza in termini di open source con l’importante esperienza e i bisogni di un amministratore pubblico.
+
+### Prossimi passi?
+
+Oggi pubblichiamo il capitolo dedicato agli sviluppatori, a cui seguiranno gli altri capitoli nei prossimi mesi.
+Ti chiediamo di leggere ed aiutarci ad evolvere il documento, solo così otterremo un risultato finale di sicura utilità. Se vorrai aiutarci ti ringraziamo fin da ora per il tuo tempo e i tuoi preziosi contributi.
```
Comment (bfabio) [inline _posts/it/2020-12-29-guida-sviluppo-gestione-software-libero.md:50]:
```suggestion
Ti chiediamo di leggere e aiutarci a evolvere il documento, solo così otterremo un risultato finale di sicura utilità. Se vorrai aiutarci ti ringraziamo fin da ora per il tuo tempo e i tuoi preziosi contributi.
```

**Review by bfabio — APPROVED** [review]
:+1:

**Pull request merged**